adapters/handlers/rest/authz/handlers_authz.go

```diff
@@ -585,7 +672,15 @@ func (h *authZHandlers) getUsersForRole(params authz.GetUsersForRoleParams, prin
 		return authz.NewGetUsersForRoleForbidden().WithPayload(cerrors.ErrPayloadFromSingleErr(err))
 	}
 
-	users, err := h.controller.GetUsersForRole(params.ID)
+	var userType models.UserTypes
+	if params.UserType == string(models.UserTypesOidc) {
```

Comment:
Will `UserTypes` ever be extended to other providers, do you think? If so, I think it would be better to use a switch-case block here rather than if-else. wdyt?

Reply:
not in the forseeable future, I can't think of any other authentication method that would make sense to add right now